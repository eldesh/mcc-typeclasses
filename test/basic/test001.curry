coin = 0 ? 1
