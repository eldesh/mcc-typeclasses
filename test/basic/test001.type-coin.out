Num _1 => _1
